src/login/logind-dbus.c

```diff
@@ -886,6 +886,7 @@ static int method_create_session(sd_bus_message *message, void *userdata, sd_bus
                 goto fail;
 
         session->type = t;
+        session->original_type = t;
```

Comment:
doesn't matter, but i'd write these two lines as `session->original_type = session->type = t;` just to emphasize that this is not a typo, but intended